docs/UserGuide.md

```diff
@@ -160,7 +163,7 @@ Examples:
 
 
 ## Company Management 
-Companies are organizations that provide job positions. Each company can have attributes such as address, billing date, and contact details, facilitating job creation and matching.
+Companies are organizations that provide job positions. Each company can have attributes such as address, billing date, and contact details, facilitating job creation and matching. A company is unique identify by its name (case-insensitive). 
```

Comment:
Last line, A company is UNIQUELY IDENTIFIED by...

Reply:
by IT'S name

Reply:
Fixed the first comment.